### PR TITLE
[JEP-211] - Update the timeline to reflect the current plan and the EPICs list

### PR DESCRIPTION
**Pull request by oleg-nenashev**
CC @jenkinsci/java11-support

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@jenkinsci/jep-developers

File: jep/211/README.adoc [hunk 1]
```diff
@@ -37,7 +37,8 @@ endif::[]
 
 | JIRA
 |
-link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - GA in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - Preview in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-51805[JENKINS-52012] - GA in weekly releases,
```
Comment (batmat) [inline jep/211/README.adoc]: Typo in second link for GA: title is wrong.
@oleg-nenashev for next time: are you OK with me directly fixing such typos in your branch, or would you rather fix things like these yourself?

File: jep/211/README.adoc [hunk 2]
```diff
@@ -220,12 +221,18 @@ The rollout procedure should be coordinated within the link:https://jenkins.io/s
...
+* Experimental Java 11 Support is available in Jenkins 2.127+
+** Announced in link:https://jenkins.io/blog/2018/06/17/running-jenkins-with-java10-11/[this blogpost]
 ** We have started integrating some patches starting from 2.127 when the “--enable-future-java” flag was introduced
-* This JEP targets Java 11 support in weekly releases.
-The plan is to announce Java 11 support when it is done, no special timing
-* LTS general availability: Java 11 support will be available in LTS once the LTS baseline updates to the Weekly release.
-** There is no plan to backport changes required for Java 11 support
+** There is no official preview announcement for weekly releases at this stage
+* link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - Preview in weekly releases
+* link:https://issues.jenkins-ci.org/browse/JENKINS-51805[JENKINS-52012] - GA in weekly releases
```
Comment (batmat) [inline jep/211/README.adoc]: same. Maybe we should not copy paste this issue list and refer to above anyway?

File: jep/211/README.adoc [hunk 3]
```diff
@@ -37,7 +37,8 @@ endif::[]
 
 | JIRA
 |
-link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - GA in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - Preview in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-51805[JENKINS-52012] - GA in weekly releases,
 link:https://issues.jenkins-ci.org/browse/JENKINS-52284[JENKINS-52284] - GA in LTS,
 link:https://issues.jenkins-ci.org/browse/JENKINS-51805[JENKINS-51805] - other non-blocker issues
```
Comment (bitwiseman) [inline jep/211/README.adoc]: @oleg-nenashev  
This is a duplicate link of the one you added.  Was that intentitional?

**Comment by oleg-nenashev** [comment]
Yes, I totally messed up the links. Should be fine now

**Pull request merged**